src/iop/colorbalancergb.c

```diff
@@ -1414,6 +1414,30 @@ void color_picker_apply(dt_iop_module_t *self, GtkWidget *picker, dt_dev_pixelpi
 }
 
 
+static void YchToRGB(dt_aligned_pixel_t *RGB_out, float chroma, float hue,
```

Comment:
Nice code refactoring. As a convention we add an underscore as prefix for static functions. Can you fix that? TIA.

Reply:
Sure. Do you mean it should be _YchToRGB or Ych_to_RGB?